intellij-plugin/build.gradle.kts

```diff
@@ -171,11 +171,7 @@ dependencies {
     zipSigner()
   }
 
-  // Coroutines must be excluded to avoid a conflict with the version bundled with the IDE
-  // See https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#coroutinesLibraries
-  implementation(project(":apollo-gradle-plugin-external")) {
-    exclude(group = "org.jetbrains.kotlinx", module = "kotlinx-coroutines-core")
-  }
+  implementation(project(":apollo-gradle-plugin"))
```

Comment:
No longer needed because it's relocated?

Reply:
The exclusion is not needed anymore because `apollo-gradle-plugin` doesn't depends on `kotlinx-coroutines` anymore. 

`apollo-gradle-plugin-tasks` does but it's a completely separate tree.

Reply:
Ah right! Cool 👍